all.json

```diff
@@ -11,6 +11,82 @@
 		"162.222.226.37",
 		"1776coin.org",
 		"1chaincrypto.com",
+		"24x7-bittreks.com",
+		"24x7-ledger.com",
+		"alienheroesworld.io",
+		"amodosecurityservices.org",
+		"app-onlinelottoclaim.us",
+		"app.anchorappweb.com",
+		"app.cosmoioapp.com",
+		"app.multiversepad.net",
+		"apps-bots.live",
+		"bonezai.com",
+		"bridge.tewrrra.money",
+		"browserhelper12.org",
+		"contentstack.store",
+		"cosmoioapp.com",
+		"dapp.liveappio.com",
+		"deal.smartokendeals.net",
+		"drinksonme.ng",
+		"elitefidelitytrade.com",
+		"event.eggroot.com",
+		"expaand.com",
+		"gate-site.xyz",
+		"giveaway-bananataskforceape.com",
+		"giveaway-blinmetaverse.com",
+		"giveaway-blueshift.com",
+		"giveaway-bscharactbit.com",
+		"giveaway-clonex.com",
+		"giveaway-cryptostake.com",
+		"giveaway-esg.com",
+		"giveaway-ezchain.com",
+		"giveaway-greenmeta.com",
+		"giveaway-helenafinancial.com",
+		"giveaway-konstrukt.com",
+		"giveaway-muesliswap.com",
+		"giveaway-paygo.com",
+		"giveaway-payzpayments.com",
+		"giveaway-pieme.com",
+		"giveaway-pleasurecoin.com",
+		"giveaway-safeukr.com",
+		"giveaway-senspark.com",
+		"giveaway-stepapp.com",
+		"giveaway-tifitoken.com",
+		"govgrantsupport.org",
+		"gpts3.gptscript.com",
+		"gptscript.com",
+		"greybull.digital",
+		"hedron.myappsio.com",
+		"liveappio.com",
+		"masterclass.binateclimited.com",
+		"meenaagro.com",
+		"mercadou.net",
+		"mints-nft.live",
+		"moneysandesh.com",
+		"moonbeaim.network",
+		"myappsio.com",
+		"navrangmontessori.org",
+		"networkbridgesvalidation.xyz",
+		"nft.binateclimited.com",
```

Comment:
Don't see anything here at this point in time...

Reply:
removing all but `networkbridgesvalidation.xyz`